src/main/java/org/haffson/adventofcode/utils/FileReaders.java

```diff
@@ -14,4 +13,18 @@
 @Component
 public class FileReaders {
 
+    public int[] readFileIntoArrayOfIntegers(String path){
+        List<String> data = new ArrayList<>();
+        InputStream inputStream = FileReaders.class.getResourceAsStream(path);
+        Scanner scanner = new Scanner(inputStream);
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            if(!line.isEmpty()){
+              data.add(line);
+            }
+        }
+        scanner.close();
+        return data.stream().mapToInt(Integer::parseInt).toArray();
```

Comment:
Macht in diesem Falle keinen großen visuellen Unterschied, aber im Allgemein bevorzugen wir es bei streams die einzelnen aufrufe untereinander zu schreiben, um es übersichtlich zu halten.

```suggestion
        return data.stream()
                   .mapToInt(Integer::parseInt)
                   .toArray();
```

Reply:
In such short stream calls, I would not enforce that suggestion though!

Reply:
True, it is more readable.